org.tigervnc.TigerVNC.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.tigervnc.TigerVNC
```

Comment:
the upstream adddate uses `org.tigervnc.vncviewer`. It has to match one way or the other.